include/psen_scan_v2/monitoring_frame_msg.h

```diff
@@ -109,6 +109,15 @@ class MonitoringFrameMsg
 
   uint32_t scan_counter_{ 0 };
   std::vector<double> measures_;
+
+public:
```

Comment:
Please, state all public stuff before the private stuff. This way the reader can easily overview the public interface of the class. Furthermore, ensure that first all public methods are stated and then all public members, etc.

I suggest that you state the possible exceptions at the beginning of the class. Please, separate the function declaration and the definition to ease readability.